ui/pages/confirmations/hooks/useNetworkClientId.ts

```diff
@@ -0,0 +1,24 @@
+import { CaipChainId } from '@metamask/utils';
+import { useCallback } from 'react';
+import { useSelector } from 'react-redux';
+
+import { getRpcDataByChainId } from '../../../../shared/modules/network.utils';
+import { getMultichainNetworkConfigurationsByChainId } from '../../../selectors';
+
+export function useNetworkClientId() {
+  const multichainNetworkConfigurations = useSelector(
+    getMultichainNetworkConfigurationsByChainId,
```

Comment:
Can we instead use the existing `selectDefaultRpcEndpointByChainId` to simplify this logic?

And since it simplifies the logic so much, could we remove this hook and just the selector directly in `useEIP7702Account`?

Reply:
PR is updated